packages/recozik-web/src/recozik_web/app.py

```
@@ -277,35 +276,32 @@
-    if not resolved_str.startswith(base_prefix):
+    # Ensure the resolved path is within the media root directory.
+    try:
+        if not resolved_path.is_relative_to(media_root):
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST, detail="Path outside media root"
+            )
+    except ValueError:
+        # This occurs on Windows if the path is on a different drive.
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST, detail="Path outside media root"
         )
```

Ruff (0.14.4)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)